homeassistant/components/flexit_bacnet/const.py

```diff
@@ -0,0 +1,31 @@
+"""Constants for the Flexit Nordic (BACnet) integration."""
+from flexit_bacnet import (
+    VENTILATION_MODE_AWAY,
+    VENTILATION_MODE_HIGH,
+    VENTILATION_MODE_HOME,
+    VENTILATION_MODE_STOP,
+    VENTILATION_MODES,
+)
+
+from homeassistant.components.climate import (
+    PRESET_AWAY,
+    PRESET_BOOST,
+    PRESET_HOME,
+    PRESET_NONE,
+)
+
+DOMAIN = "flexit_bacnet"
+
+VENTILATION_TO_PRESET_MODE_MAP = {
+    VENTILATION_MODES[VENTILATION_MODE_STOP]: PRESET_NONE,
+    VENTILATION_MODES[VENTILATION_MODE_AWAY]: PRESET_AWAY,
+    VENTILATION_MODES[VENTILATION_MODE_HOME]: PRESET_HOME,
+    VENTILATION_MODES[VENTILATION_MODE_HIGH]: PRESET_BOOST,
```

Comment:
This feels weird to me - not saying it is wrong, but may be worth taking another look at it to see if you have a better approach.

from a readability standpoint, its just a bit confusing what the actual keys are

Reply:
It did feel weird and it's not simplified.